exporters/metric/cortex/example/go.mod

```diff
@@ -2,11 +2,6 @@ module go.opentelemetry.io/contrib/exporters/metric/cortex/example
 
 go 1.14
 
-replace (
-	go.opentelemetry.io/contrib/exporters/metric/cortex => ../
-	go.opentelemetry.io/contrib/exporters/metric/cortex/utils => ../utils/
-)
```

Comment:
These replace statements are critical to include for development. When there are unreleased changes to the `go.opentelemetry.io/contrib/exporters/metric/cortex` and `go.opentelemetry.io/contrib/exporters/metric/cortex/utils` packages and we want to update the example prior to a release they are needed to refer to the correct and local code. Additionally, when we are making a release we will bump the version of these packages prior to them actually existing so the build needs to rely on the local code instead of a non-existent tagged remote release.

Can we update the example in a way where these are preserved?

Reply:
I guess we could map the parent directories into the container. I can give it a go.